src/main/java/io/confluent/connect/hdfs/HdfsSinkConnectorConfig.java

```diff
@@ -246,7 +247,21 @@
 
   }
 
-  public HdfsSinkConnectorConfig(Map<String, String> props) {
+  public static HdfsSinkConnectorConfig getConfig(Map<String, String> props) {
+    return getConfig(CONFIG_DEF, props);
+  }
+
+  protected static HdfsSinkConnectorConfig getConfig(
+      ConfigDef configDef,
+      Map<String, String> props
+  ) {
+    ConcurrentMap<String, String> propsCopy = new ConcurrentHashMap<>(props);
+    propsCopy.putIfAbsent(StorageCommonConfig.STORAGE_CLASS_CONFIG, HdfsStorage.class.getName());
+    propsCopy.putIfAbsent(HdfsSinkConnectorConfig.FORMAT_CLASS_CONFIG, AvroFormat.class.getName());
+    return new HdfsSinkConnectorConfig(configDef, propsCopy);
+  }
+
+  protected HdfsSinkConnectorConfig(Map<String, String> props) {
```

Comment:
Yeah, you reminded me why the voice in my head was telling me _never add a static method in Config again_ ... I'll fix the `public static ConfigDef getConfig() {` which by the way contains a wrong blacklist now.